source/documentation/submit-draft-movement.html.md.erb

```diff
@@ -46,7 +47,7 @@ A [list of potential IE815 error codes](error-codes.html#ie815-submit-draft-move
 
 For level 1 and level 2 validation, you will receive an instant response.
 
-For level 3 validation, once you have submitted a movement, you will need to poll the [Get all messages for an excise movement using the movement ID](/api-documentation/docs/api/service/excise-movement-control-system-api/1.0/oas/page#tag/Movements/operation/getAllMessages) endpoint. You will receive one of the following:
+For level 3 validation, once you have submitted a movement, you will need to poll the [Get all messages for an excise movement using the movement ID](/documentation/get-all-messages.html) endpoint. You will receive one of the following:
```

Comment:
is this file (get-all-messages.html) have a link to the spec api? if not should be better leave this so they can see the endpoint from the API Spec. If yes, ignore my comment.

Reply:
There is a link to the API spec for the Submit Draft Movement endpoint earlier on this page, and a link to the API spec for the GetAllMessages at the start of the Get Messages page